XenAdmin/Wizards/NewNetworkWizard.cs

```diff
@@ -60,6 +61,7 @@ public partial class NewNetworkWizard : XenWizardBase
         private readonly NetWDetails pageNetworkDetails;
         private readonly NetWBondDetails pageBondDetails;
         private readonly NetWChinDetails pageChinDetails;
+        private readonly NetWSriovDetails pageSriovDetails;

```

Comment:
Some implementation is missing from here as well as the NetWSriovDetails page: how is the wizard collecting the info from that page and how is it using it when Finish is clicked?

Reply:
Yes, this PR is just about the implementation of GUI. Because the XAPI part hasn't been ready, so the details will update in the furture PR. Thanks.